.gitlab-ci.yml

```diff
@@ -28,7 +28,7 @@ flake8:
   stage: build
   image: pipelinecomponents/flake8:f087c4c
   variables:
-    FLAKE8_FILES: "tests/*.py"
+    FLAKE8_FILES: "tests/*.py tools/beerocks_analyzer/conf/conf1.py tools/beerocks_analyzer/*.py tools/commends/*.py"
```

Comment:
```suggestion
    FLAKE8_FILES: "tests/*.py tools/beerocks_analyzer/conf/conf1.py tools/commands/*.py"
```

Reply:
beerocks_analyzer already passes the code check.
typo commEnds->commAnds is addressed in https://github.com/prplfoundation/prplMesh/pull/1118